### DISCUSSION TITLE
Is it possible add a feature that shows where a protein falls on the Atlas using the 'Look up in the Atlas' tool?

### DISCUSSION THREAD

**Discussion by Loganz97** [facebookresearch/esm#429] [Ideas]
As part of a project, I am exploring the sequence space of enzymes similar to the ones I am working with. However, what would make my job significantly easier is if I could use the atlas to find nearest neighbors to a protein of interest,

Thanks in advance!

**Comment by tomsercu** [comment]
See https://esmatlas.com/resources?action=search_sequence
Search Sequence or Search Structure using MMseqs and foldseek respectively

**Reply by Loganz97** [reply]
Thanks! This shows me close neighbors but I want to see where they fall in the visualization specifically.

**Reply by tomsercu** [reply]
Ah yes gotcha, unfortunately we don't provide that functionality. Note that the search sets of the high confidence structures is only 37M proteins, which still contains many proteins which aren't part of the atlas visualization.

**Reply by Loganz97** [reply]
Thanks for the quick response! I guess the main benefit of being able to see close neighbors on the atlas, is it is possible to relatively quickly identify new (interesting) morphologies by just looking at close neighbors on the atlas. 

When im looking for new enzymes of interest it's really time consuming to try and identify novel morphologies for enzymes that have similar functions. Current tools like Blast and FoldSeek show you things that are similar, not things that are a bit different.
